DataManagement/GitTutorial.Rmd

```diff
@@ -166,26 +177,37 @@ knitr::include_graphics("Pics/gittutorial/14_git.png")
 ```
 <br/><br/>
 <br/><br/>
-Enter the project directory name, paste the url link from above and create the project.
+Paste the url link from GitHub.
+It should be something like https://github.com/myaccount/myrepo.git
+Enter the **Project directory name**.
+Again choose a useful name that describes the project.
+Choose where to store the project and click **Create Project**.
 <br/><br/>
 ```{r paste-url, out.width = '50%'}
 knitr::include_graphics("Pics/gittutorial/15_paste_url.png")
 ```
 <br/><br/>
 <br/><br/>
+Now you are ready to start coding.
 
 
 ## Push existing RStudio project to GitHub
 
-The general workflow for how to start a new project is described above (see create and clone repository). 
-But when you start to work with GitHub you will have RStudio projects that you want to connect to GitHub. Here is how it works.
-
+The general workflow for how to start a new project is described above (see 0.5 Create a GitHub repository). 
```

Comment:
Should be able to create a cross-reference to 0.5 (v similar to method for cross-reference to figures)

Reply:
Yes, I don't know how but it must be possible

Reply:
https://bookdown.org/yihui/bookdown/cross-references.html